core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -322,6 +323,25 @@ class NavigationHelpersClass {
     return expandedList;
   }
 
+  storeCollapsedSuperCategoriesState(key, value) {
```

Comment:
Can you add unit tests for these new methods? Please use existing 'navigation-helpers.spec.js' file for this purpose

Reply:
Added 😸